Volume/Analytics/Events.md

```diff
@@ -18,7 +20,7 @@
 # Parameters
 | Name | Slug | Description | Values |
 |---|---|---|---|
-| publicationID | publication | Publication UUID | |
+| publicationSlug | publication | Publication unique shortened name | |
```

Comment:
just curious, what is the purpose of publication unique shortened name?

Reply:
The issues with publicationIDs is that they are automatically generated by MongoDB and assigned differently between root-level publications and publications embedded within article objects. So we have manually assigned slugs instead that are more developer-friendly but still unique.

Reply:
i'm not sure what the difference between publicationSlug and Publication unique shortened name though?

Reply:
`publication unique shortened name` is the documented description of `publicationSlug`. If you compile the `Events.md` file from Markdown into PDF you'll see a table that shows that info more clearly.